py4j-java/src/test/java/py4j/instrumented/InstrPythonClient.java

```diff
@@ -48,9 +48,10 @@ public class InstrPythonClient extends PythonClient {
 
 	public InstrPythonClient(Gateway gateway, List<Class<? extends Command>> customCommands, int pythonPort,
 			InetAddress pythonAddress, long minConnectionTime, TimeUnit minConnectionTimeUnit,
-			SocketFactory socketFactory, Py4JJavaServer javaServer, boolean enableMemoryManagement, int readTimeout) {
+			SocketFactory socketFactory, Py4JJavaServer javaServer, boolean enableMemoryManagement, int readTimeout,
+			int createSocketTimeout) {
```

Comment:
```suggestion
			int connectTimeout) {
```

and 🔭 constructor for API back compat